homeassistant/components/hydrawise/entity.py

```diff
@@ -70,3 +70,9 @@ def _handle_coordinator_update(self) -> None:
         self.controller = self.coordinator.data.controllers[self.controller.id]
         self._update_attrs()
         super()._handle_coordinator_update()
+
+    @property
+    def available(self) -> bool:
+        """Set the entity availability."""
+        # Default availability is controller is online and last update success
+        return self.controller.online and self.coordinator.last_update_success
```

Comment:
```suggestion
        return super().available and self.controller.online
```

Reply:
Clever, did not realize the super class already checks last_update_success